docs/domains/index.html

```diff
@@ -1279,7 +1287,7 @@
           "type": "string"
         },
         "introspectVersion": {
-          "description": "If present, every time this value is updated, the operator will start introspect domain job",
+          "description": "If the domain is running, the operator will restart its introspector job when this value is changed.This field is ignored when the domainHomeSourceType source type is FromModel.",
```

Comment:
Typo missing space after the period -> ".  This field is ignored when the domainHomeSourceType source type is FromModel."